.github/README.md

```diff
@@ -54,9 +54,25 @@ The Oppia Android code is released under the [Apache v2 license](https://github.
 
 ## Keeping in touch
 
-  * [Blog](https://medium.com/oppia-org)
   * [Discussion forum](http://groups.google.com/group/oppia)
   * [Announcements mailing list](http://groups.google.com/group/oppia-announce)
-  * Social media: [YouTube](https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw), [FB](https://www.facebook.com/oppiaorg), [Twitter](https://twitter.com/oppiaorg)
 
 We also have public chat rooms on Gitter: [https://gitter.im/oppia/oppia-android](https://gitter.im/oppia/oppia-android). Drop by and say hello!
+
+## Social Media
+
+<a href="https://twitter.com/oppiaorg">
+  <img align="left" alt="Oppia's Twitter" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/twitter.svg" />
+</a>
```

Comment:
Why not use Markdown here, instead? I think something odd is happening with the anchor tags since the preview is showing that all of the icons are going to twitter rather than their respective locations. Can you try to switch this to Markdown both for consistency, and to see if it fixes the issue? Or, do we have to specify the size of the SVGs?

Reply:
Looks like I will keep this open for a while. Solved the all twitter link, but need to look for markdown flavour, as required 2 links one is the URL where the click will hop and other is the icon link.

Reply:
I think the current approach is fine--Markdown would be preferred, but given the constraints I think HTML is reasonable.